### PR TITLE
Sync with the equivalent ruby solution

### PR DESCRIPTION
**Pull request by m-Peter**
cc @chief @astathopoulos

### CODE REVIEW

**Comment by chief** [comment]
:+1: nice

File: src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java [hunk 1]
```diff
@@ -146,8 +146,8 @@ private int rule0(char s[], int len) {
 
     if (len > 6 && (endsWith(s, len, "σκαγιου"))
         || endsWith(s, len, "σκαγιων")
-        || endsWith(s, len, "ολογιου")
-        || endsWith(s, len, "ολογιων")
+        // || endsWith(s, len, "ολογιου")
+        // || endsWith(s, len, "ολογιων")
```
Comment (astathopoulos) [inline src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java]: Please remove the commented lines completely. We have git history to look for them if we want.

Comment (m-Peter) [inline src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java]: Done.

**Comment by astathopoulos** [comment]
:+1:

**Comment by m-Peter** [comment]
Can you have a look again? I missed couple things:
- Forgot to run the test suite
- Forgot to update 'ημερολογια'

**Comment by astathopoulos** [comment]
@m-Peter I prefer to split this commit to a single commit for each corresponding change. I know that we should keep the two implementation in sync, but the repos are independent from each other. I can't understand which change affected the corresponding stemming samples.

**Comment by m-Peter** [comment]
@astathopoulos Are the commits separated correctly now?

**Comment by astathopoulos** [comment]
@m-Peter Yes, much better! :+1:

**Comment by m-Peter** [comment]
Is there anything left to do?

**Comment by astathopoulos** [comment]
@m-Peter Fine by me. You already have my :+1:

**Pull request merged**